### PR TITLE
Add column movement UI

### PR DESCRIPTION
**Pull request by tripolkaandrey**
This PR adds UI for column movement. Here is a demo:
demo

Also, subscriptions to events are removed in favour of subscribing to changes in the `Board` projection's state.

### CODE REVIEW

File: web/client-js/src/store/board/actions/command/move-column-action.ts [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+  to: ColumnPosition;
+};
+
+/**
+ * Moves a column.
+ */
+export default class MoveColumnAction extends BoardAction<
+  MoveColumnPayload,
+  void
+> {
```
Comment (armiol) [inline web/client-js/src/store/board/actions/command/move-column-action.ts:57]: Can we please leave the closing bracket on the previous line?

File: web/client-js/src/views/BoardView.vue [hunk 2]
```diff
@@ -1,10 +1,29 @@
...
-    <KanbanColumn
-      v-for="(column, $columnIndex) of board.getColumnList()"
-      :key="$columnIndex"
-      :column="column"
-    />
+    <div id="columns">
+      <div
+        v-for="(column, $columnIndex) of board.getColumnList()"
+        :key="$columnIndex"
+      >
```
Comment (armiol) [inline web/client-js/src/views/BoardView.vue:7]: Can we please have the corresponding bracket of a tag on the same line as the last significant text?

This changeset illustrates the poor readability with the standalone `>`, `/>`, etc, bouncing all over the places.

**Review by armiol — CHANGES_REQUESTED** [review]
@tripolkaandrey please see my comments.

Other than those, LGTM.

**Review by armiol — APPROVED** [review]

**Comment by alexander-yevsyukov** [comment]
@tripolkaandrey,  @armiol, what if we start applying [Aqua](https://www.jetbrains.com/aqua/) to our example projects?

**Pull request merged**